README.md

```diff
@@ -23,23 +23,22 @@ Normally Routers retrieve the data for their Virtual JSON resource from backend
 First we create a folder for our application server.
 
 ~~~bash
-mkdir falcor-app-server
-cd falcor-app-server
+mkdir falcor-app-server && cd !$
 npm init
 ~~~
 
 Now we install the Falcor Router.
 
 ~~~bash
-npm install falcor-router --save
+npm i falcor-router -S
```

Comment:
`-S` = `--save`. Let's show the world we love NPM and know it top to bottom :smile_cat: